factcast-store/src/main/java/org/factcast/store/internal/filter/blacklist/PgBlacklistDataProvider.java

```diff
@@ -46,16 +44,17 @@ public PgBlacklistDataProvider(
   @Override
   public void afterSingletonsInstantiated() {
     bus.register(this);
-    updateBlacklist();
+    updateBlacklist(); // just to be safe
```

Comment:
Actually I think its necessary otherwise the readonly instance would operate on an empty blacklist until the first change notification comes in, isnt it?